types/plotly.js/index.d.ts

```diff
@@ -1584,7 +1584,7 @@ export interface Config {
      * function to add the background color to a different container
      * or 'opaque' to ensure there's white behind it
      */
-    setBackground: () => string | 'opaque' | 'transparent';
+    setBackground: ((gd: PlotlyHTMLElement, bgColor: string) => any) | 'opaque' | 'transparent';
```

Comment:
```suggestion
    setBackground: ((gd: PlotlyHTMLElement, bgColor: string) => void) | 'opaque' | 'transparent';
```
any particular reason to have this working differently compared to underlying functions?
https://github.com/plotly/plotly.js/blob/12069023b46b4583259a7cfba295e1e653c15266/src/plot_api/plot_api.js#L412-L423

Reply:
Nope, nothing except my own ignorance & fear of causing breakage. The previous type def made it look to me like the function was expected to return a string, but in fact it doesn't have to (and looks like the return value isn't used at all) so I was broadening to `any`, but on second thought that could be misleading. If someone can easily change then feel free. Otherwise I will update this PR later when I'm at home (where I have the full repo cloned and don't have to justify the time to my employer :D)